README.md

```diff
@@ -1,9 +1,8 @@
-# @hyperapp/logger
+# `hyperapp-logger`
```

Comment:
# <img height=24 src=https://cdn.rawgit.com/JorgeBucaran/f53d2c00bafcf36e84ffd862f0dc2950/raw/882f20c970ff7d61aa04d44b92fc3530fa758bc0/Hyperapp.svg> Hyperapp Logger

Reply:
What do you think?

Reply:
https://gist.github.com/JorgeBucaran/533f75c2330cf79afc2e622ad7a07e84

Reply:
#lgtm